src/elements/core/styles/core.scss

```diff
@@ -200,6 +207,18 @@ input[data-sbb-time-input] {
   }
 }
 
+// TODO: Move back to the teaser-hero components when the global css refactoring happens
+:is(sbb-teaser-hero) {
+  sbb-image {
```

Comment:
nothing new, but in for teaser hero wouldn't the "img" be supported?

Reply:
It was missing :/ 
I also added visual spec for them

Reply:
very nice, thank you